01-Lab/SoftwareIntro.ipynb

```diff
@@ -4,253 +4,459 @@
+    "\n",
+    "A very common neuroimaging package is FSL. FSL has viewer that we can open in the bash terminal, and it has some very helpful features for understanding more about images.\n",
+    "\n",
+    "__We'll first see how to:__ <br>\n",
+    "<br>\n",
+    "__(a) learn basic information about the image attributes__ <br> \n",
+    "-What kind of image is this <br>\n",
+    "-What is the voxel size <br>\n",
+    "-Others? <br>\n",
+    "<br>\n",
+    "__(b) look at the image histogram.__ <br>\n",
+    "An image histogram shows the distribution of all the intensity values that make up the image. Looking at these is helpful to see what intensities are generally “brain” versus background and to get comfortable with the idea that images are a collection of voxels with different intensity values (e.g., see Handbook Figure 2.1, p14). The intensity values mean something about what tissue we *think* we’re seeing based on the physics of MRI. It is also helpful to remember that the basis of most image processing tools is signal processing and statistics on these intensity values to improve the detection of the \"signal\" we're interested in (e.g., brain structure or activity) from noise (e.g., motion). <br>\n",
+    "-What is the range of values in our sub-97-T1w_defaced.nii.gz image <br>\n",
+    "-Is the histogram multi-modal? What would that tell us?  <br>\n",
+    "-What are the typical values for gray matter, white matter, and CSF? <br>\n",
+    "<br>\n",
+    "__(c) look at our functional image and its timeseries.__ <br>\n",
+    "-Compare and contrast the image intensities of white matter and gray matter from the structural with the functional image. <br>\n",
+    "-Show the image timeseries <br>\n",
+    "-Show the movie of volumes over time, how many dimensions is the file? <br>\n",
+    "-Compare the spatial resolution of the structural and functional <br>\n",
+    "-Can we overlay the functional on top of the structural image? <br>"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "anat_filename"
+    "%%bash\n",
+    "\n",
+    "#the viewer version we'll use is called fslview, you can open it without opening an image by typing fslview\n",
+    "#fslview\n",
+    "pwd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Image meta-data\n",
+    "\n",
+    "In addition to the information in the image, another important aspect of data with the images is their meta-data (Handbook, p14). This information is often stored in a [header] (http://nipy.org/nibabel/nibabel_images.html#the-image-header) that is tagged onto the image file itself or as a separate file. For nifti images the header information is kept tagged onto the image itself, so it is a handy to know how to look at the header for images on the command line. We'll see how to do this with fsl and nibabel below."
```

Comment:
remove space between `[header]` and `(http://nipy.org/nibabel/nibabel_images.html#the-image-header)`